frontend/src/components/BookPage.js

```diff
@@ -135,77 +129,65 @@ class BookPage extends Component {
       <DataGrid
         localeText={this.dataGridLocaleText()}
         rows={
-            this.props.bookOrders.filter(order => order.type == this.props.type || this.props.type == 2)
-            .filter(order => order.currency == this.props.currency || this.props.currency == 0)
-            .map((order) =>
-            ({id: order.id,
-              avatar: window.location.origin +'/static/assets/avatars/' + order.maker_nick + '.png',
-              robot: order.maker_nick,
-              robot_status: order.maker_status,
-              type: order.type ? t("Seller"): t("Buyer"),
-              amount: order.amount,
-              has_range: order.has_range,
-              min_amount: order.min_amount,
-              max_amount: order.max_amount,
-              currency: this.getCurrencyCode(order.currency),
-              payment_method: order.payment_method,
-              price: order.price,
-              premium: order.premium,
-            })
```

Comment:
I consider this serialization unnecessary, we already have an `book` object with all the information, I see no point on just changing names and confuse future developers